config/routes.rb

```diff
@@ -58,5 +59,8 @@
     get 'works/thumbnail/:id' => 'works#thumbnail'
     get 'artists/commissions/:id' => 'artists#commissions'
     get 'requests/request_exist/:search_params' => 'requests#request_exist'
+    put 'artists/lock_user/:id' => 'artists#lock_user'
```

Comment:
What's the difference of `put`?

Reply:
Put request is necessary since the controller method `lock_user` updates the `locked_at` field for devise to properly lock a user.